scipy/stats/tests/test_distributions.py

```diff
@@ -3505,6 +3527,12 @@ def test_cdf(self):
 
         assert_almost_equal(stats.skellam.cdf(k, mu1, mu2), skcdfR, decimal=5)
 
+    def test_extreme_mu2(self):
+        # check that crash reported by gh-17916 large mu2 is resolved
+        x, mu1, mu2 = 0, 1, 4820232647677555.0
+        assert_allclose(stats.skellam.pmf(x, mu1, mu2), 0, atol=1e-16)
+        assert_allclose(stats.skellam.cdf(x, mu1, mu2), 0, atol=1e-16)
```

Comment:
```suggestion
        assert_allclose(stats.skellam.cdf(x, mu1, mu2), 1, atol=1e-16)
```
I didn't realize the support of `skellam` was `-inf, inf`. When `mu1 = 1`, the mode of the distribution appears to be near `-mu2`, so all the probability mass is to the left, and the CDF at `x=0` is `1`.